### PR TITLE
add image uploading section

### PR DESCRIPTION
**Pull request by rzig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,27 @@ Want to edit something you see on the website? Below is a (hopefully-updated) ma
...
+
+- Use a descriptive name
+- Compress images first (use [Squoosh](https://squoosh.app))
+
+Some dont's:
+
+- Delete images that are still in use (use Ctrl + F to find uses of a filename)
+- Delete `logo.svg`, ever
+- Delete anything that starts with `favicon` or is not an image file.
+
```
Comment (alex-m-zhang) [inline README.md:77]: What if I want to

**Review by alex-m-zhang — APPROVED** [review]
LGTM

**Review by Liseiden — APPROVED** [review]
LGTM

**Comment by lnguy7** [comment]
Pog PR

**Pull request merged**